Update Collections docs (#7369)
PrefectHQ/prefect: docs/collections/usage.md

---

> description: Prefect Collections provide pre-built tasks and flows that help you build workflows quickly.

Reviewer: ```suggestion
description: Prefect Collections provide Prefect integrations that help you build dataflows quickly.
```

tags:
    - tasks
    - flows
    - blocks
    - collections
    - task library
    - contributing
---

# Using Collections

> ## Installing Collections

Reviewer: ```suggestion
## Installing a Collection
```

To use a Prefect Collection, first install the collection via `pip`.

As an example, to use `prefect-aws`:

```bash
pip install prefect-aws
```

> ## Registering Blocks within Collections

Reviewer: ```suggestion
## Registering Blocks from a Collection
```

> Then, register to view the blocks on Prefect Cloud:

Reviewer: ```suggestion
Once the Prefect Collection is installed, [register the blocks](concepts/blocks/#registering-blocks-for-use-in-the-prefect-ui) within the collection to view them in the Prefect Cloud UI:

As an example, to register the block in `prefect-aws`:
```

```bash
prefect block register -m prefect_aws.credentials
prefect block register -m prefect_aws.ecs
```

> ## Running Tasks within Collections

Reviewer: ```suggestion
## Using Tasks and Flows from a Collection
```

> The AWS tasks and flows in that collection can then be imported and called within your flow:

Reviewer: ```suggestion
Collections also contain pre-built tasks and flows that can be imported and called within your code.

As an example, to read a secret for AWS Secrets Manager with the `read_secret` task:
```

---
description: Prefect Collections provide Prefect integrations that help you build dataflows quickly.
tags:
    - tasks
    - flows
    - blocks
    - collections
    - task library
    - contributing
---

# Using Collections

## Installing a Collection

To use a Prefect Collection, first install the collection via `pip`.

As an example, to use `prefect-aws`:

```bash
pip install prefect-aws
```

## Registering Blocks from a Collection

> Once the Prefect Collection is installed, [register the blocks](concepts/blocks/#registering-blocks-for-use-in-the-prefect-ui) within the collection to view them in the Prefect Cloud UI:

Reviewer: ```suggestion
Once the Prefect Collection is installed, [register the blocks](/concepts/blocks/#registering-blocks-for-use-in-the-prefect-ui) within the collection to view them in the Prefect Cloud UI:
```

> As an example, to register the block in `prefect-aws`:

Reviewer: ```suggestion
As an example, to register the blocks available in `prefect-aws`:
```

```bash
prefect block register -m prefect_aws.credentials
prefect block register -m prefect_aws.ecs
```

## Using Tasks and Flows from a Collection

Collections also contain pre-built tasks and flows that can be imported and called within your code.

> As an example, to read a secret for AWS Secrets Manager with the `read_secret` task:

Reviewer: ```suggestion
As an example, to read a secret from AWS Secrets Manager with the `read_secret` task:
```
